docs/units/index.rst

```diff
@@ -168,6 +168,14 @@ Using `astropy.units`
    constants_versions
    conversion
 
+Acknowledgments
+===============
+
+This code is adapted from the `pynbody
```

Comment:
it is not really "this code" any more, right? What about rephrasing that the "first implementation was adapted from..." or something similar?

Reply:
Maybe just "started from" or "was originally based on"

Reply:
Yes, good catch. Thank you both!